crates/typst-library/src/html/mod.rs

```diff
@@ -17,21 +17,21 @@ pub fn module() -> Module {
     Module::new("html", html)
 }
 
-/// An HTML element that can contain Typst content.
+/// Typstのコンテンツを含むことができるHTML要素。
 ///
-/// Typst's HTML export automatically generates the appropriate tags for most
-/// elements. However, sometimes, it is desirable to retain more control. For
-/// example, when using Typst to generate your blog, you could use this function
-/// to wrap each article in an `<article>` tag.
+/// TypstのHTMLエクスポートは、ほとんどの要素に対して適切なタグを自動的に生成します。
+/// ただし、場合によっては、より強いコントロールを手元に残しておくことが望ましい場合があります。
+/// 例えば、Typstを使ってブログを生成する場合、
+/// それぞれの記事を`<article>`タグで囲むためにこの関数を使用できます。
```

Comment:
nitpickぽいですが、

```suggestion
/// この関数を用いると、それぞれの記事を`<article>`タグで囲めます。
```

の方が自然な感じがします。